conformance/utils/http/http.go

```diff
@@ -344,14 +344,27 @@ func setRedirectRequestDefaults(req *roundtripper.Request, cRes *roundtripper.Ca
 		expected.RedirectRequest.Host = cRes.RedirectRequest.Host
 	}
 
-	if expected.RedirectRequest.Port == "" {
-		expected.RedirectRequest.Port = req.URL.Port()
-	}
-
 	if expected.RedirectRequest.Scheme == "" {
 		expected.RedirectRequest.Scheme = req.URL.Scheme
 	}
 
+	if expected.RedirectRequest.Port == "" {
```

Comment:
If it's redirecting to `https` scheme on port `80` it should definitely include the port though, since `https` should only skip the port if it's `443`.

Reply:
Agree with Nick's explanation. The test seems to be working as expected.

Reply:
I think it has been updated since my original comment. But I am seeing other failures so I would like some time to validate if Istio impl is broken or if the tests are unexpected. Will update today